package.json

```diff
@@ -8,12 +8,13 @@
     "scripts": {
         "build": "webpack -p --config ./webpack.config.js",
         "start": "NODE_ENV=production node ./src/main.js",
-        "dev": "DEBUG=knex:tx NODE_ENV=development nodemon --inspect src/main.js",
+        "dev": "DEBUG=knex:tx NODE_ENV=development DEBUG='appstore*' nodemon --watch ./src --watch ./client/ --inspect src/main.js",
```

Comment:
You could add the `appstore*` to the first `DEBUG` variable so you don't overwrite the `knex:tx`:

```
DEBUG=knex:tx,appstore*
```